### ISSUE TITLE
Filtered Tiddler lists and templates don't render properly

### ISSUE THREAD

**Issue by Plloi** [Jermolene/TiddlyWiki5#3121]
Description:
Filtered lists piped into templates behave differently than a single tiddler, I suspect this is in error. In the example linked below I have a Tiddler affected by this bug, and one what I assume is the correct output but with the items added one at a time manually.

Example of issue: [via rawgit](https://rawgit.com/Plloi/940fdaae5f1fe3e18d30af0dc1c84c3f/raw/BuggedWiki.html)

**Comment by Jermolene** [comment]
Hi @Plloi thanks, it indeed looks like there's a problem here: specifically, the transcluded tiddlers are being transcluded in inline mode, rather than blockmode (hence why the block more formatting within them isn't displaying correctly). I'll investigate.

**Comment by pmario** [comment]
This syntax: `{{{ [tag[incme]] || IncTemplate}}}` can't be used here, since it only produces inline transclusions. 

If `IncTemplate` looks like this, it works: 

```
''<$transclude field=title mode=block />''
<$transclude mode=block />
```

The following code does also work. 

```
<$list filter="[tag[incme]]">

;{{!!title}}

{{!!text}}
</$list>
```

**Comment by pmario** [comment]
I did modify the first post.

**Comment by pmario** [comment]
imo can be closed.

**Comment by Plloi** [comment]
yeah, if the template pipe i was was using is supposed to produce an inline instead of a block transclusion this can be closed - not a bug
